config/testgrids/default.yaml

```diff
@@ -20,7 +20,7 @@ default_test_group:
 
 default_dashboard_tab:
   open_test_template: # The URL template to visit after clicking on a cell
-    url: https://prow.k8s.io/view/gcs/<gcs_prefix>/<changelist>
+    url: https://prow.k8s.io/view/gs/<gcs_prefix>/<changelist>
```

Comment:
per @fejta 
>btw should now be gs not gcs
>
>Yes:
>https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-test-infra-deploy-prow/1364279379974164480
>
>Deprecated:
>https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/post-test-infra-deploy-prow/1364279379974164480
>
>(gs is the official scheme for gcs)
>
>(previously gcs was meant to distinguish from "query the api server rather than blob storage")